### DISCUSSION TITLE
Realtime Inspector feedback

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#19596] [General]
We're excited to announce the release of the Realtime Inspector: a new tool designed to help prototype, inspect, and debug Realtime messages. We launched this feature during LWX in December 2023, and would love to get your feedback on it.

What do you like most about it? Least?
Are there any features you find particularly useful? 
Anything you think could be improved? 
Anything you think is missing?
 
Happy to hear your thoughts, suggestions, and experiences.

**Comment by techopswhiz** [comment]
Main thing is I've never been able to get it to work with self-hosted realtime-only.
